dom/src/main/scala/org/scalajs/dom/Document.scala

```diff
@@ -124,10 +125,11 @@ abstract class Document extends Node with NodeSelector with DocumentEvent with P
   /** In an HTML document creates the specified HTML element or HTMLUnknownElement if the element is not known. In a XUL
     * document creates the specified XUL element. In other documents creates an element with a null namespaceURI.
     */
-  def createElement(tagName: String): Element = js.native
+  def createElement(tagName: String, options: String | ElementCreationOptions = js.native): Element = js.native
```

Comment:
I've been thinking about it for a bit, and I think instead of union types we should prefer overloads where possible. It's a bit more verbose but they tend to work better with inference. I'm going to add this to the contributing guide.

```suggestion
  def createElement(tagName: String): Element = js.native
  def createElement(tagName: String, options: String): Element = js.native
  def createElement(tagName: String, options: ElementCreationOptions): Element = js.native
```

Reply:
The verbosity isn't bad at all!